_includes/pbc/all/install-features/202307.0/install-the-dynamic-store-prices-feature.md

```diff
@@ -0,0 +1,517 @@
+
+### 1) Set up configuration
+
+{% info_block warningBox "Configuration stores.php" %}
+
+Since the dynamic store is now enabled, configuration for the store is stored in the database, making the file `config/Shared/stores.php` deprecated. 
+
+The default store configuration will now be imported using new data import modules such as `CurrencyDataImport`. These modules will populate the store configuration in the database.
+A new major module `Currency`, have been introduced and that is responsible for extending store data and configuring it in the database. You can find migration guide [here](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/install-and-upgrade/update-the-currency-module.html)
+
+{% endinfo_block %}
+
+
+### 2) Set up the database schema and transfer objects
+
+1. Adjust the schema definition so entity changes trigger events:
+
+**src/Pyz/Zed/Currency/Persistence/Propel/Schema/spy_currency.schema.xml**
+
+```xml
+
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed"
+          xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd"
+          namespace="Orm\Zed\Currency\Persistence" package="src.Orm.Zed.Currency.Persistence">
+
+    <table name="spy_currency_store" idMethod="native">
+        <behavior name="event">
+            <parameter name="spy_currency_store_all" column="*"/>
+        </behavior>
+    </table>
+
+</database>
+
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied by adding a currency to the store in the Backoffice interface. 
+
+{% endinfo_block %}
+
+2. Run the following commands to apply database changes and generate entity and transfer changes:
+
+```bash
+console propel:install
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied by checking your database:
+
+| DATABASE ENTITY                       | TYPE   | EVENT   |
+|---------------------------------------|--------|---------|
+| spy_store.fk_currency                 | column | added   |
+| spy_currency_store                    | table  | added   |
+
+{% endinfo_block %}
+
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied in transfer objects:
+
+| TRANSFER | TYPE | EVENT | PATH |
+| --- | --- | --- | --- |
+| CurrencyCriteria | class | created | src/Generated/Shared/Transfer/CurrencyCriteriaTransfer  |
+| Store.defaultCurrencyIsoCode     | property | added | src/Generated/Shared/Transfer/CustomerTransfer  |
+
+{% endinfo_block %}
+
+### 3) Configure export to Redis
+
+1.  Set up publisher plugins:
+
+| PLUGIN | SPECIFICATION | PRERQUISITES | NAMESPACE |
+| --- | --- | --- | --- |
+| CurrencyStoreWritePublisherPlugin | Publishes currency store data to storage table. | None | Spryker\Zed\StoreStorage\Communication\Plugin\Publisher\CurrencyStore |
```

Comment:
currency and store data?
currency-store data?

Reply:
Store's currencies